.github/workflows/build-assets.yml

```diff
@@ -54,7 +56,9 @@ jobs:
           echo "internal_tag=$(echo $community_tag | sed 's/community/internal/')" >> $GITHUB_OUTPUT
           community_base_ref=${{ github.event.base_ref || github.ref }}
           echo "community_branch=$(echo $community_base_ref | cut -d'/' -f3)" >> $GITHUB_OUTPUT
-          echo "cmake_options=-DCMAKE_BUILD_TYPE=RelWithDebInfo -DVCPKG_FILES_DIR=/hpcc-dev -DCPACK_THREADS=0 -DUSE_OPTIONAL=OFF" >> $GITHUB_OUTPUT
+          echo "cmake_options=-DCMAKE_BUILD_TYPE=RelWithDebInfo -DVCPKG_FILES_DIR=/hpcc-dev -DCPACK_THREADS=0 -DUSE_OPTIONAL=OFF -DSIGN_MODULES=ON" >> $GITHUB_OUTPUT
+          echo 'mount_ecl_signing_key=source="${{ github.workspace }}/private.key",target=/hpcc-dev/private.key,type=bind,consistency=cached' >> $GITHUB_OUTPUT
```

Comment:
Do we want to create a new mount?  It could be dropped into the build mount?

Reply:
Good idea, dropping the key into folder_build and the load_key will pull from the mount_build target location inside the container. Testing it now https://github.com/Michael-Gardner/HPCC-Platform/actions/runs/4057581726/jobs/6983501947